src/adk/api/remote_api.py

```diff
@@ -724,6 +724,41 @@ def experiments_list(self) -> List[ExperimentType]:
 
         return experiment_list
 
+    def __get_backend_type(self, backend_type: str) -> List[BackendTypeType]:
+        """
+        Get the backend info for the remote backend in the experiment data.
+
+        Returns:
+            0 or 1 backend types with this type
+        """
+        backend_types = self.__qne_client.list_backendtypes()
```

Comment:
Actually I don't get this function here at all:

1. I cannot find list_backendtypes() anywhere in the API Client, but I assume it returns a list of BackendTypes
2. Later when you call self.__get_backend_type() you expect it to return a Backend object (since you check the OFFLINE status, which is for Backends, not for BackendTypes)

Reply:
1. line 441 of qne_client.py
2. Nope, you were right with the previous remark. It returns a backend type. Calling it backend was wrong.
The OFFLINE in backendtype is not the status of a single backend but the status of all the backends for this backend_type (summed up). Only when all backends for this backend type are OFFLINE, then this status for backend_type goes to OFFLINE. (See api-router, backend_type model property status)